styles/utilities.css

```
@@ -53,17 +53,20 @@
   max-width: 100px;
 }
 
+.rm-list-style {
```

```suggestion
.rm-list-style[role=list] {
```
To make sure the attr is not omitted.